src/tests/__snapshots__/Components.test.js.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Test for the components should render the cardComponent 1`] = `
+<div
+  className="card-item"
+>
+  <div
+    className="active-data"
+  >
+    <div
+      className="top"
+    >
+      <h3 />
+    </div>
+  </div>
+  <div
+    className="inactive-data"
+  >
+    <h5>
+      Total Population
+    </h5>
```

Comment:
Kindly ensure all your app test cases are passing for a more professional codebase.
![Screenshot from 2023-08-12 19-05-26](https://github.com/ctokoli/react-capstone/assets/44624138/4217553f-91cf-4762-b85f-fb4061223ee7)